src/boot.tsx

```diff
@@ -22,6 +22,22 @@ export default (element: HTMLDivElement, hub: Hub) => {
     applyMiddleware(createEpicMiddleware(rootEpic))
   );
 
+  const getPackages = (packages: string) => {
+    if (packages){
+      return "\nfrom dcl_package_manager import install_packages, get_registry; install_packages(["+formatPackages(packages)+"], get_registry('/registry/registry.json'))";
```

Comment:
I think we can use a string template here. Something like this might work (disclaimer I haven't tested this)

```ts
return `
    from dcl_package_manager import install_packages_from_registry
    install_packages_from_registry([${formatPackage(packages)}]
`;
```

I think we should also expose a higher level function from our manager that internally combines `install_packages` and `get_registry`. This way we keep our registry config inside our module and container.

Reply:
Yes will do this, although I'd keep the exposed function (the one that combines `install_packages` and `get_registry`) named as `install_packages()` and change the name of the internal install function. Also, if I put the string on multiple lines, python gives indentation errors. So I might do as you suggested but keep it as a one liner